docs/make_index.py

```diff
@@ -55,6 +55,7 @@ def write_index_rst(readme_file=None, write_file=None):
         "   fast_calh5_meta\n"
         "   utility_functions\n"
         "   developer_docs\n"
+        "   conventions\n"
```

Comment:
I think this should go higher up. Maybe right after the tutorial?

Reply:
Bumped up in a4b4bfa160f5d0bb923b28970bb457c333554526